src/Swagger.php

```diff
@@ -205,4 +209,12 @@ public function getSecurityDefinitions()
     {
         return $this->securityDefinitions;
     }
+
+    /**
+     * @return array
+     */
+    public function getParameters()
```

Comment:
No setter?

As a side note, this change requires a new minor. Not a problem for me, I will just release `0.3.0`.